tools/update-defs.py

```
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```

This implementation seems overcomplicated for such a simple string-processing task.

---

Does this shorter one seem more appropriate? I don't want to make it too simple, because I want to make sure it will error out rather than mangle the generated file or silently drop data.